test/boost/sstable_3_x_test.cc

```diff
@@ -45,6 +45,7 @@
 #include "readers/combined.hh"
 
 #include <boost/range/algorithm/sort.hpp>
+#include <fmt/ranges.h>
```

Comment:
Why this include?

Reply:
explained in the commit message. BTW, this is the first tiny step to deprecate our homebrew solutions provided by `to_string.h` in favor of fmtlib's generic formatter for ranges and tuple.